tests/test_protocols/test_abci/test_abci.py

```diff
@@ -171,6 +171,7 @@ def build_message(self) -> AbciMessage:
             version="0.1.0",
             block_version=1,
             p2p_version=1,
+            abci_version="0.35.7",
```

Comment:
not relevant here, but this is actually `0.17.0` I believe

Reply:
I think this is not the version of the protocol, it's the version of the abci app afaik. Please check